packages/picasso/src/Avatar/Avatar.tsx

```diff
@@ -44,9 +45,10 @@ const isBrowserSupportsObjectFit = 'objectFit' in document.documentElement.style
 const renderLogo = ({
   classes,
   src,
-  size
-}: Pick<Props, 'src' | 'size'> & JssProps) => {
-  if (!src || !size || ['small', 'xsmall', 'xxsmall'].includes(size)) {
+  size,
+  name
+}: Pick<Props, 'src' | 'size' | 'name'> & JssProps) => {
+  if (!src || !size || !name || ['small', 'xsmall', 'xxsmall'].includes(size)) {
```

Comment:
This if condition is not clear to me. What do you think about this?

```suggestion
  const isEmpty =  !src || !size || !name
  const isTooSmall = ['small', 'xsmall', 'xxsmall'].includes(size)

  if (isEmpty || isTooSmall) {
```